web/middleware/json_params.py

```diff
@@ -9,11 +9,14 @@
 
 
 async def json_params(request):
+    logger.debug(request.headers)
+    logger.debug(request.body)
     if (request.headers.get('content-type', None) == 'application/json'):
+        if (request.body == b'' or request.body == ''):
```

Comment:
could be reformatted as `if (len(request.body) == 0):`

Reply:
should be fixed